### DISCUSSION TITLE
Tooltips inside nested elements

### DISCUSSION THREAD

**Discussion by andrewpitts15** [floating-ui/floating-ui#3006] [React]
I have created a tooltip component based on the example at https://codesandbox.io/p/sandbox/xenodochial-grass-js3bo9 and have some nested components each with their own tooltip.

```
<Tooltip>
    <TooltipTrigger asChild={true}>
        <ParentComponent>
            <Tooltip>
                <TooltipTrigger asChild={true}>
                    <ChildComponent />
                </TooltipTrigger>
                <TooltipContent>
                    Child Tooltip
                </TooltipContent>
            </Tooltip>
        </ParentComponent>
    </TooltipTrigger>
    <TooltipContent>
        Parent Tooltip
    </TooltipContent>
</Tooltip>
```

When I hover over the child component, I would like it so only the child component's tooltip appears, however it current displays both tooltips. How should I modify the component so only the lowermost tooltip appears?

**Comment by seedydocloop** [comment]
I have the same need.

I tried to implement this via a controlled open state, but it triggers rerender of my children, which breaks UI.
